clouddq/runners/dbt/dbt_runner.py

```diff
@@ -213,15 +220,42 @@ def _resolve_dbt_path(
             logger.debug(f"Using 'dbt_path': {dbt_path}")
         return dbt_path
 
-    def _prepare_dbt_project_path(self) -> None:
+    def _prepare_dbt_project_path(self,
+                                  intermediate_table_expiration_hours: Optional[int]) -> None:
+
         dbt_project_path = self.dbt_path.absolute().joinpath("dbt_project.yml")
-        if not dbt_project_path.is_file():
+        if intermediate_table_expiration_hours:
```

Comment:
if 'intermediate_table_expiration_hours' is not optional then we don't need this.

Reply:
please remove this if clause and reduce indent for everything after it